packages/playground/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "sdk-playground",
```

Comment:
Whats the point of putting the playground inside the packages directory? vs apps where it was before?

Reply:
thanks for the review @cb-jake 
no good reason. i thought these arguments sounded reasonable, and couple of open source libraries that i took a look also had this approach, so went with its recommended path.
should we reintroduce distinction of apps vs packages?
![image](https://github.com/user-attachments/assets/ee5d854f-aa15-4d58-806f-f7d914cde8da)

Reply:
@cb-jake moved it back to /apps

Reply:
Nice either or are fine I was curious if there was a special reason for the directory change, i.e. CI, etc...